test/Microsoft.AspNet.Mvc.FunctionalTests/ModelBindingTest.cs

```diff
@@ -2159,7 +2159,7 @@ public async Task
             // Assert
             Assert.Equal(HttpStatusCode.NoContent, response.StatusCode);
             var body = await response.Content.ReadAsStringAsync();
-            Assert.Equal(string.Empty, body);
+            Assert.Equal("null", body);
```

Comment:
Discussed in person, the action code is specifically creating an `ObjectResult() { Value = null }`